ubuntudesign/tutorials.ubuntu.com
# Build tooling

Set up initial build tooling.

This currently uses a simple config file with IDs per line, a yaml config may be a better choice to switch to but will require switching away from bash as the language (It has limited yaml support).


Done:
- Added script to build codelabs from IDs in config file
- Added script to change img src to absolute paths, to revisit at a later time.
- Removed imported codelabs content from the codebase
- Added example content

QA:
- Check site loads with example content
- Use `./bin/build-tutorials` and check content imported correctly
  - You may need to auth by trying to add one manually first: `./tools/codelabs add [GOOGLE_DOC_ID]`
